Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -1273,6 +1272,11 @@ bool WebPageProxy::hasSameGPUAndNetworkProcessPreferencesAs(const API::PageConfi
     return !updateSharedPreferencesForWebProcess(sharedPreferences, configuration.preferences().store());
 }
 
+bool WebPageProxy::hasSameGPUAndNetworkProcessPreferencesAs(const WebPageProxy& page) const
+{
+    return hasSameGPUAndNetworkProcessPreferencesAs(Ref { page }->protectedConfiguration());
```

Comment:
The `Ref { }` should be needed here since `page` is a parameter and we expect the caller to protect it.

Reply:
It's needed because `protectedConfiguration()` is not inline. I tried inlining that too but we can't.